superset-frontend/src/dashboard/stylesheets/dnd.less

```diff
@@ -123,7 +129,12 @@
   }
 
   & > .empty-droptarget:first-child {
-    height: 24px;
+    height: 48px;
     margin-top: -24px;
+    margin-bottom: -24px;
+  }
+
+  & > .empty-droptarget:only-child {
+    height: 80vh;
```

Comment:
This `vh` unit might not be accurate on viewports that are very tall or very short. The header area is not a flexible 20%, but some fixed height I believe. You might be better off with something like `height: calc(100vh - XXpx)`

That said I'm LOVING this idea.

Reply:
Thanks a lot! I'm also concerned about this `vh`. On the other hand top hasn't defined size and in mobile view it becomes really big. But I will try figure something out.

Reply:
Not sure if it would work (depends on the positioning of its ancestors), but `position:absolute` with top/bottom/left/right of 0 might work.